plugins/extractors/elastic/elastic.go

```diff
@@ -101,22 +101,30 @@ func (e *Extractor) Extract(ctx context.Context, emit plugins.Emit) (err error)
 		return
 	}
 	x := reflect.ValueOf(r["indices"]).MapRange()
+	var indexes []string
```

Comment:
do we need this `indexes`? we can directly process the index in the `for x.Next()` block right?

Reply:
yes, we can do that. removed all the changes from elastic.go btw, made them just to sort the index and columns. but they are not necessary anymore.